changelog.d/3445.bugfix

```diff
@@ -0,0 +1 @@
+do not use six.itervalues() on SortedDict()
```

Comment:
The changelog is broadly aimed at server admins, so could this be something more like "Fix bug in sending presence over federation" or something?